Rocket.Chat/Resources/pt-PT.lproj/Localizable.strings

```diff
@@ -63,8 +63,8 @@
 "alert.email_verification.title" = "Verificação de e-mail";
 "alert.email_verification.message" = "Enviamos um e-mail para você confirmar o seu registo. Se você não receber um e-mail em breve, volte e tente novamente.";
 
-"alert.discarding_changes.title" = "Mudanças não salvas";
-"alert.discarding_changes.message" = "As suas alterações não salvas foram descartadas.";
+"alert.discarding_changes.title" = "Alterações não guardadad";
```

Comment:
guardadad isn't a typo?

Reply:
hell yeah. 😂
can you change it to `guardadas` or you want me to do it?
Currently on the move and in the phone.

Cheers